docs/developer/available-plugins/openid_connect.mdx

```diff
@@ -315,10 +318,19 @@ A successful response:
 }
 ```
 
-
 ## Saleor as a resource server
 
-### User authentication flow
+In resource server flow, Saleor API only verifies if token included in request is valid. Operations like login, logout or token refresh have to be handled directly by auth provider API.
```

Comment:
In resource server flow, Saleor API only verifies if the token included in the request is valid. Operations like login, logout or token refresh have to be handled directly by auth provider API.